### ISSUE TITLE
[Request] PNG video codec for MOV file format

### ISSUE THREAD

**Issue by CombatAura** [lay295/TwitchDownloader#38]
Hello,
Thank you for your hard work man, your program is amazing!

but yeah, this is just a request. It seems like certain video editors are pretty picky with how they want MOV files. and it seems like MOV with the PNG video codec can work with video editors like Cyberlink PowerDirector and Camtasia to show the chat with the transparent background on.

Oh and sorry to annoy you with another one, but if there's a way to have the chat rendered video length the same as the vod length. That be super awesome!

But yeah, you've done already enough to even have this whole thing possible so it's all good if you can't. Appreciate the hard work you put in!

**Comment by lay295** [comment]
Have you tried ProRes? ProRes files are already massive, and I imagine that lossless PNG video codec would be even more massive.

And length of the chat and vod will be fixed in the next version, have just been lazy about getting around to it.

**Comment by CombatAura** [comment]
> Have you tried ProRes? ProRes files are already massive, and I imagine that lossless PNG video codec would be even more massive.
> 
> And length of the chat and vod will be fixed in the next version, have just been lazy about getting around to it.

Yeah, i tried ProRes, it unfortunately didn't work. I only know that the PNG video codec works for MOV files is because I use ffmpeg ffe to re encode the MOV file from RLE to PNG.

ProRes files are really big, for an 1Hr and 6min chat vod, the size was almost 9GB. So i don't really plan on using that one much unless for something really specific.

For ffe, you can control the bitrate of the video you're re encoding, so I just set it as around the same average bitrate as the rendered out chat vod used from your program. So I actually wouldn't know if the PNG video codec would be massive or not, it's really not a big thing for me, re encoding doesn't really take a lot of time for me.
